pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -889,24 +956,18 @@ func aceToRules(aclArgs types.AppNetworkACLArgs, ace types.ACE) (types.IPTablesR
 		}
 
 		if aclRule3.RuleID != -1 {
-			for _, uplink := range aclArgs.UpLinks {
-				aclRule3.Table = "mangle"
-				aclRule3.Chain = "PREROUTING"
-				if aclArgs.NIType == types.NetworkInstanceTypeSwitch {
-					aclRule3.Rule = append(aclRule3.Rule, "-m", "physdev",
-						"--physdev-in", uplink)
-				}
-				aclRule3.IsMarkingRule = true
-				chainName := fmt.Sprintf("%s-%s-%d",
-					aclArgs.BridgeName, aclArgs.VifName, aclRule3.RuleID)
-
-				// Embed App id in marking value
-				markingValue := (aclArgs.AppNum << 24) | aclRule3.RuleID
-				createMarkAndAcceptChain(aclArgs, chainName, markingValue)
-				aclRule3.Action = []string{"-j", chainName}
-				aclRule3.ActionChainName = chainName
-				rulesList = append(rulesList, aclRule3)
-			}
+			aclRule3.Table = "mangle"
```

Comment:
Does the change anything for the mesh type where we can have multiple uplinks?

Reply:
This code is specific for Switch network instance type.

Reply:
OK